packages/dd-trace/src/gateway/channels.js

```diff
@@ -0,0 +1,8 @@
+'use strict'
+
+const dc = require('./dc')
+
+module.exports = {
+  INCOMING_HTTP_REQUEST_START: dc.channel('dd-trace:INCOMING_HTTP_REQUEST_START'),
+  INCOMING_HTTP_REQUEST_END: dc.channel('dd-trace:INCOMING_HTTP_REQUEST_END')
```

Comment:
These aren't the only two channels you've made, are they?

Reply:
I thought there were other channels and that for whatever reason only two of them were being included here. It seem I'm mistaken though, so nevermind.